pmlc/conversion/gpu_to_spirv/gpu_to_spirv.cc

```diff
@@ -75,7 +165,10 @@ struct GPUToSPIRVCustomPass
 void populateStdxToSPIRVPatterns(MLIRContext *context,
```

Comment:
I think we need a separate pattern named populateStdxToSPIRVGLSLpatterns and put all lowering that requires GLSLOps in it. When it is a opencl-spirv module, we should not convert the ops to GLSLOps.